javascript/ql/src/Security/CWE-200/PrivateFileExposure.qhelp

```diff
@@ -5,20 +5,33 @@
 
 <overview>
   <p>
-    Placeholder
+    Libraries like <code>express</code> provide easy methods for serving entire 
+    directories of static files from a web server.
+    However, using these can sometimes lead to accidential information exposure. 
+    If for example the <code>node_modules</code> folder is served, then an attacker
+    can access the <code>_where</code> field from a <code>package.json</code> file, 
+    which gives the attacker access to the absolute path of the file. 
   </p>
 </overview>
 
 <recommendation>
   <p>
-    Placeholder
+    Limit which folders of static files are served from a web server.
   </p>
 </recommendation>
 
 <example>
   <p>
-    Placeholder
+    In the example below all the files from the <code>node_modules</code> are served. 
+    This allows clients easy access to all files inside that folder, but also allows
```

Comment:
Suggesting:
This not only allows clients to easily access all the files inside that folder, but also to access potentially private information inside <code>package.json</code> files.

Reply:
Yeah, that sentence could use a rewrite. 

Hmm. The `package.json` file is one of the files in the folder (that wasn't very clear from my original sentence). 
How about: 
```
This allows clients to easily access all the files inside that folder,
which includes potentially private information inside package.json files.
```